typeql/src/main/java/org/ldbcouncil/snb/impls/workloads/typeql/operationhandlers/TypeQLSingletonOperationHandler.java

```diff
@@ -0,0 +1,50 @@
+package org.ldbcouncil.snb.impls.workloads.typeql.operationhandlers;
+
+import org.ldbcouncil.snb.driver.DbException;
+import org.ldbcouncil.snb.driver.Operation;
+import org.ldbcouncil.snb.driver.ResultReporter;
+import org.ldbcouncil.snb.impls.workloads.typeql.TypeQLDbConnectionState;
+import org.ldbcouncil.snb.impls.workloads.operationhandlers.SingletonOperationHandler;
+
+import com.vaticle.typedb.client.api.TypeDBSession;
+import com.vaticle.typedb.client.api.answer.ConceptMap;
+import com.vaticle.typedb.client.api.TypeDBTransaction;
+
+import java.text.ParseException;
+import java.util.Iterator;
+import java.util.Map;
+
+public abstract class TypeQLSingletonOperationHandler<TOperation extends Operation<TOperationResult>, TOperationResult>
```

Comment:
I don't know if Singleton / List  is the best way to divide the OperationHandlers.
Can't you use the ListOperationHandler to answer Singleton queries as well?

We'll likely have to define a separate OperationHandler for each result type (`Stream<ConceptMap>, Stream<ConceptMapGroup>, NumericGroup, Numeric`). Let's wait to see if fetch does aggregation before doing this.